app/src/main/java/org/p2p/wallet/striga/repository/StrigaKycDataLocalRepository.kt

```diff
@@ -0,0 +1,9 @@
+package org.p2p.wallet.striga.repository
+
+import org.p2p.wallet.striga.repository.model.StrigaOccupation
+import org.p2p.wallet.striga.repository.model.StrigaSourceOfFunds
+
+interface StrigaKycDataLocalRepository {
```

Comment:
how's kyc data sounds? it is too ambiguous?

Reply:
maybe StrigaValuesLocalRepo?

Reply:
StrigaValuesLocalRepo or even StrigaPresetDataLocalRepo sounds better, as this repo doesn't actually contains any KYC-specific data, it's just a dataset